doc/news/changes/minor/20200214KatrinMang

```diff
@@ -0,0 +1,4 @@
+New: a split_by_block() function is added which partitions the set indices for example based on a block structure into blocks.
+IndexSet (*this) should be defined with respect to the global block structure. In particular, it should be the same for all MPI processes.
```

Comment:
This should be a short description since entries for the whole library are collected here. 

Just move this sentence to the description in `include/deal.II/base/index_set.h` (possibly dropping "this"). Also, it is `dofs_per_block` that should be defined consistently across MPI  ranks.